tests/test_tea.py

```diff
@@ -17,33 +18,98 @@ def ExampleModel(self):
         param_nco.update(nco_input_data, check_already_exists=False)
         tea_nco = TEA(param_nco, nco_input_data)
         return tea_nco
+    
+    # test stack breakdown
+    def test_stack_breakdown(self):
+
+        # load example model
+        tea_nco = self.ExampleModel()
+
+        # check if pandas dataframe and dict
+        self.assertIsInstance(tea_nco.stack_breakdown, dict)
+        self.assertIsInstance(tea_nco.stack_breakdown_dataframe, pd.DataFrame)
 
-    # test stack energy densities dict
+        # import stack breakdown dataframe from csv
```

Comment:
it might be cleaner to have the reference data frames just defined at the top of this file rather than reading from csv. happy to leave it as is for now though